quinn-proto/src/connection.rs

```diff
@@ -1889,10 +1889,13 @@ impl Connection {
         // To ensure that sufficient data is available for sampling, packets are padded so that the
         // combined lengths of the encoded packet number and protected payload is at least 4 bytes
         // longer than the sample required for header protection.
-        if let Some(padding) =
-            (space.header_crypto.sample_size() + 4).checked_sub(buf.len() - header_len + pn_len)
+        let protected_payload_len = (buf.len() + AEAD_TAG_SIZE) - header_len;
+        if let Some(padding_minus_one) =
```

Comment:
Surely there's some semantic reason for `padding_minus_one`? :grin:

Reply:
It's that or have it come out as zero sometimes and execute the body gratuitously, which would be harmless as it was previously written, but now that there's a logging statement I figured I'd make the branch a little more robust.

Reply:
I just meant the variable name.

Reply:
Whoops, that was actually computing padding *plus* one. Fixed.